tiatoolbox/models/engine/patch_predictor.py

```diff
@@ -250,7 +251,13 @@ def __init__(
 
         self.ioconfig = ioconfig  # for storing original
         self._ioconfig = None  # for storing runtime
-        self.model = model
+        self.model = (
+            compile_model(  # for runtime, such as after wrapping with nn.DataParallel
+                model,
+                mode=rcParam["torch_compile_mode"],
```

Comment:
Do we need rcparam for this? We can just set this as kwargs argument in the engines.